src/nft/components/collection/TraitSelect.tsx

```diff
@@ -113,7 +113,10 @@ export const TraitSelect = ({ traits, type, index }: { traits: Trait[]; type: st
   const debouncedSearch = useDebounce(search, 300)
 
   const searchedTraits = useMemo(
-    () => traits.filter((t) => t.trait_value.toString().toLowerCase().includes(debouncedSearch.toLowerCase())),
```

Comment:
FYI this is fixing a breaking error. the collection page for art blocks could not be shown without this fix (page crashing error).

Reply:
imo should be in a separate pr in future